src/library_webgpu.js

```diff
@@ -1314,11 +1317,9 @@ var LibraryWebGPU = {
 
     function makeVertexBuffer(vbPtr) {
       if (!vbPtr) return undefined;
-
       return {
         "arrayStride": {{{ gpu.makeGetU64('vbPtr', C_STRUCTS.WGPUVertexBufferLayout.arrayStride) }}},
-        "stepMode": WebGPU.VertexStepMode[
-          {{{ gpu.makeGetU32('vbPtr', C_STRUCTS.WGPUVertexBufferLayout.stepMode) }}}],
+        "stepMode": WebGPU.VertexStepMode[{{{ gpu.makeGetU32('vbPtr', C_STRUCTS.WGPUVertexBufferLayout.stepMode) }}}],
```

Comment:
Sorry, I was trying to get through this review before I left for vacation, but now that you've [noticed](https://dawn-review.googlesource.com/c/dawn/+/97900/comments/67b68f64_24899918) the thing about it not being option 0, I'm confused about what this does. If we set `stepMode` to `undefined` then we get the default  value from WebIDL, which is `"vertex"`. I think if `stepMode` is `VertexBufferNotUsed`, then we want `makeVertexBuffer` to return `null`, not `{ arrayStride, stepMode: undefined, attributes }`.

Reply:
Ah.. rechecked spec and fixed. Thanks for catching this!

Reply:
Oh I see what you mean on comment at https://dawn-review.googlesource.com/c/dawn/+/97900. Yeah in this case it doesn't make more sense for undefined jsrepr

Reply:
But given it's the library_webgpu_enum_tables, 
```
VertexStepMode: [
      'vertex',
      'instance',
      undefined,
    ],
```
It's different from VertexStepMode enum value . It's a matter of how emscripten impl interprets the enum value.
I think it's still okay just to keep it as undefined for generated info.